editor.planx.uk/src/@planx/components/Pay/Public/Pay.tsx

```diff
@@ -101,11 +109,17 @@ function Component(props: Props) {
   const handleError = useErrorHandler();
 
   useEffect(() => {
-    if (isNaN(fee) || fee <= 0) {
-      // skip the pay component because there's no fee to charge
+    // Auto-skip component when fee=0
+    if (fee <= 0) {
       return props.handleSubmit({ auto: true });
     }
 
+    // If props.fn is undefined, display & log an error
+    if (isNaN(fee)) {
```

Comment:
Whilst making this change could we also visually handle this a little more gracefully in the component which displays the value?

<img width="765" alt="image" src="https://github.com/theopensystemslab/planx-new/assets/20502206/3e30f8ae-2c19-42a2-9e55-7934743ea7a4">

Reply:
Yeah I didn't think much of this because it should really _never_ be seen :sweat_smile:

Now updated to display "Unknown": 
![Screenshot from 2023-10-06 10-43-28](https://github.com/theopensystemslab/planx-new/assets/5132349/c1f950ea-753c-40ee-8b0d-1cd97cae3c4c)